README.md

```diff
@@ -74,17 +77,25 @@ this method should set the pizza's temperature to be 300. Note: you may not use
 the variable pizza inside your method.
 
 Your Answer:
-```js
-// write code here
+```
+var pizza = {
+    temperature: "70",
+    bake: function (){
+        temperature = 300;
+    }
+}
 ```
```

Comment:
When setting the temperature to 300 inside `bake`, you need to reference the temperature using `this.temperature`.